sycl/source/detail/thread_pool.hpp

```diff
@@ -30,13 +30,16 @@ class ThreadPool {
   std::mutex MJobQueueMutex;
   std::condition_variable MDoSmthOrStop;
   std::atomic_bool MStop;
+  std::atomic_uint MJobsInExecution;
```

Comment:
```suggestion
  std::atomic_uint MNumOfJobs;
```
Or maybe `MJobsInPool`

Reply:
fixed in https://github.com/intel/llvm/pull/6837/commits/71e9048af88ddcc853968cf9bbaa050f63ad323e